src/formatting.rs

```diff
@@ -97,7 +98,9 @@ fn format_project<T: FormatHandler>(
         directory_ownership.unwrap_or(DirectoryOwnership::UnownedViaBlock),
         !input_is_stdin && !config.skip_children(),
     )
-    .visit_crate(&krate)?;
+    .visit_crate(&krate)?
+    .into_iter()
+    .filter(|(_, module)| !contains_skip(module.attrs()));
```

Comment:
Actually I think I changed my mind sorry. Instead of having filtering scattered about in multiple places, I'd like to consolidate it one (there's several other checks on lines 114-119). It would _probably_ be best to add filtering iterations after line 102/105 so that the reported duration of parsing is as closely confined to the parsing pass as possible, but I don't feel too strongly about it.